node-module-system/add.js

```diff
@@ -0,0 +1,3 @@
+export default function add(x, y) {
+  return parseInt(x) + parseInt(y);
```

Comment:
It would be easier to do the parsing in calculate.js so you could assume that these are already numbers in this function.